core/src/main/java/org/stellar/anchor/sep24/Sep24Transaction.java

```diff
@@ -4,118 +4,259 @@
 
 @SuppressWarnings("unused")
 public interface Sep24Transaction {
+
+  /**
+   * The database ID.
+   *
+   * @return The generated database ID.
+   */
   String getId();
 
   void setId(String id);
 
-  String getDocumentType();
-
-  void setDocumentType(String documentType);
-
+  /**
+   * Unique, anchor-generated id for the transaction.
+   *
+   * @return The <code>id</code> field of the SEP-24 transaction history.
+   */
   String getTransactionId();
```

Comment:
It doesn't make much sense to me why we have both an ID and a transactionId. Do we really need both?

Reply:
This was to separate the database ID from the SEP24 transaction ID. In some cases, the SEP24 transaction ID cannot be used as a database ID.

Reply:
In the code do we use different ones or are we defaulting to use the same in both fields?

Or is this defined in the Anchor side?